04_health/exposure-to-trauma/exposure-to-trauma-2020.Rmd

```diff
@@ -0,0 +1,434 @@
+  scatter_grid()
+
+```
+
+```{r}
+deaths %>%
+  ggplot(aes(deaths, rate_injury_death)) +
+  geom_point(alpha = 0.2) +
+  labs(title = "High Injury Death Rates are Concentrated in Counties with Few Injury Deaths") +  
+  scatter_grid()
+
+```
+
+
+## 5. Save the data
+This section reads in the Urban Institute's universe of counties, joins the CDC injury death data to those counties, and writes out the file.
+
+Read in Urban county file for 2014-2020
+```{r}
+all_counties <- read.csv(here::here("04_health", "county-populations.csv")) %>%
+  filter(!year %in% c(2021, 2022)) %>%
+  mutate(state = str_pad(state, 2, pad = "0"),
+         county = str_pad(county, 3, pad = "0"),
+         GEOID = str_c(state, county))
+
+```
+
+16 observations (4 counties) are in our universe of counties but not in the CDC data.
+```{r}
+missing_CDC <- anti_join(x = all_counties, y = deaths, by = "GEOID")
+
+missing_CDC %>%
+  group_by(state_name, county_name) %>%
+  count()
+
+```
+
+49 observations (7 counties) are in the CDC data but not our universe of counties.
+```{r}
+missing_urban <- anti_join(x = deaths, y = all_counties, by = "GEOID")
+
+missing_urban %>%
+  group_by(county_name) %>%
+  count()
+
+```
+
+Data for Bedford city, VA (FIPS code 51515) are only available for 2013 and earlier; for computational purposes, counts and population estimates for 2014 and later are set to zero. 
+
+Death rates for Clifton Forge city, VA (FIPS code 51560) are available only for 1999-2000. For 2001 and beyond, death counts and population estimates for Clifton Forge city have been combined with those for Alleghany county, Virginia (FIPS code 51005).
+
+Data for Prince of Wales-Outer Ketchikan Census Area, AK (FIPS code 02201), Skagway-Hoonah-Angoon Census Area, AK (FIPS code 02232) and Wrangell-Petersburg Census Area, AK (FIPS code 02280) are only available for 2013 and earlier; for computational purposes, counts and population estimates for 2014 and later are set to zero.
+
+Shannon County, SD was renamed to Oglala Lakota County, SD in 2014 (ratified early 2015) ([source](https://en.wikipedia.org/wiki/Oglala_Lakota_County,_South_Dakota).
+```{r}
+deaths <- deaths %>%
+  mutate(
+    county_name = if_else(county_name == "Shannon County, SD", "Oglala Lakota County, SD", county_name),
+    GEOID = if_else(GEOID == "46113", "46102", GEOID)
+  )
+
+```
+
+Wade Hampton Census Area, AK was renamed to Kusilvak Census Area, AK in 2015.
+```{r}
+deaths <- deaths %>%
+  mutate(
+    county_name = if_else(county_name == "Wade Hampton Census Area, AK", "Kusilvak Census Area, AK", county_name),
+    GEOID = if_else(GEOID == "02270", "02158", GEOID)
+  )
+
+```
+
+Chugach Census Area, AK and Copper River Census Area, AK are still missing mortality data from CDC. In early 2019, Valdez-Cordova Census Area, AK was abolished and replaced by the Chugach Census Area and the Copper River Census Area. 
+    todo(): not sure if we already have a way to crosswalk this
```

Comment:
I implemented this for the age-adjusted rate. My understanding is that this approach won't work for interpolating the confidence interval or standard error because the population is smaller in each of Chugach Census Area and Copper River Census Area than in Valdez-Cordova Census Area.

Reply:
Thanks for trying this! I should have thought this through more carefully before making this suggestion; I don't think this is working correctly. I think the fundamental challenge is that the metric we're reporting (death rates) is standardized by population. So when we interpolate population (count) and deaths (count) and then calculate the deaths rates for each of the new census areas... we're just back at the original death rate we started with. My suggestion would be to just set the death rate for these new census areas to be the same and to set the data quality flag for these observations as low since I'm not sure what else we can do here. But I'll link us up with Aaron in a thread to double check.